test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs

```diff
@@ -714,5 +714,91 @@ public void UninstallDeprecatedPackageFromUI(ProjectTemplate projectTemplate, st
             // Assert
             CommonUtility.AssertUninstalledPackageByProjectType(VisualStudio, projectTemplate, project, packageName, XunitLogger);
         }
+
+        [StaFact]
+        public void InstallTopLevelPackageHavingTransitiveFromUI()
+        {
+            // Arrange
+            EnsureVisualStudioHost();
+            var dte = VisualStudio.Dte;
+            var solutionService = VisualStudio.Get<SolutionService>();
+            solutionService.CreateEmptySolution();
+            var project = solutionService.AddProject(ProjectLanguage.CSharp, ProjectTemplate.NetCoreClassLib, "TestProject");
+            VisualStudio.ClearOutputWindow();
+            solutionService.SaveAll();
+
+            // Act
+            CommonUtility.OpenNuGetPackageManagerWithDte(VisualStudio, XunitLogger);
+            var nugetTestService = GetNuGetTestService();
+            var uiwindow = nugetTestService.GetUIWindowfromProject(project);
+            uiwindow.InstallPackageFromUI("Moq", "4.20.69");
```

Comment:
Moq version >= 4.20.0 is blocked by internal compliance tooling. Please use a different package. Ideally one that is owned by Microsoft (lower risk of the packge owner making changes that cause it to get blocked), and has little chance of having a known vulnerability reported in the future. This will reduce the maintenance efforts required in the future. If the test generated its own test package to install, there'd be almost zero risk of compliance tooling causing maintenance efforts.

Reply:
> Moq version >= 4.20.0 is blocked by internal compliance tooling. Please use a different package. Ideally one that is owned by Microsoft (lower risk of the packge owner making changes that cause it to get blocked), and has little chance of having a known vulnerability reported in the future. This will reduce the maintenance efforts required in the future. If the test generated its own test package to install, there'd be almost zero risk of compliance tooling causing maintenance efforts.

Ok, thanks for the reminder, I'll change the package.